Mobile Buy SDK/Mobile Buy SDK/Utils/BUYApplePayHelpers.m

```diff
@@ -51,11 +51,6 @@ @interface BUYApplePayHelpers ()
 
 @implementation BUYApplePayHelpers
 
-- (instancetype)initWithClient:(BUYClient *)client checkout:(BUYCheckout *)checkout
-{
-	return [self initWithClient:client checkout:checkout shop:nil];
-}
-
 - (instancetype)initWithClient:(BUYClient *)client checkout:(BUYCheckout *)checkout shop:(BUYShop *)shop
```

Comment:
Whether to use ivar or property notation for private read/write properties is a convention. However, the standard practise is moving towards the latter (property/setter) access. I've personally flip-flopped on this. I do believe that ivar access is correct for `-init` methods, but we should use setters/property access elsewhere. There are a few reasons, including KVO.